cos/static/toporgs.json

```diff
@@ -1,194 +1,206 @@
-  "Organization": "Astrophysics Source Code Library"
- },
- {
-  "Organization": "Authorea"
- },
- {
-  "Organization": "Behavioral Science & Policy Association"
- },
- {
-  "Organization": "Berkeley Initiative for Transparency in the Social Sciences"
- },
- {
-  "Organization": "Bio, Tech and Beyond"
- },
- {
-  "Organization": "Busara Center for Economics"
- },
- {
-  "Organization": "Carnegie Mellon University"
- },
- {
-  "Organization": "Center for Open Science"
- },
- {
-  "Organization": "Committee on Publication Ethics (COPE)"
- },
- {
-  "Organization": "DataONE"
- },
- {
-  "Organization": "Databrary"
- },
- {
-  "Organization": "Dataverse"
- },
- {
-  "Organization": "Dryad"
- },
- {
-  "Organization": "ECS - The Electrochemical Society"
- },
- {
-  "Organization": "EGAP"
- },
- {
-  "Organization": "F1000"
- },
- {
-  "Organization": "Figshare"
- },
- {
-  "Organization": "Foundation for Earth Science"
- },
- {
-  "Organization": "Frontiers"
- },
- {
-  "Organization": "Global Biological Standards Institute"
- },
- {
-  "Organization": "ICPSR"
- },
- {
-  "Organization": "Innovations for Poverty Action"
- },
- {
-  "Organization": "International Association for Relationship Research"
- },
- {
-  "Organization": "Laura and John Arnold Foundation"
- },
- {
-  "Organization": "MDPI (Multidisciplinary Digital Publishing Institute)"
- },
- {
-  "Organization": "Milestones Research, LLC"
- },
- {
-  "Organization": "Mozilla Science Lab"
- },
- {
-  "Organization": "NeCTAR (National eResearch Collaboration Tools and Resources)"
- },
- {
-  "Organization": "Open Atmospheric Society"
- },
- {
-  "Organization": "Open Context"
- },
- {
-  "Organization": "Open Therapuetics"
- },
- {
-  "Organization": "OpenfMRI"
- },
- {
-  "Organization": "Overleaf"
- },
- {
-  "Organization": "Paperity"
- },
- {
-  "Organization": "PeerJ"
- },
- {
-  "Organization": "Pensoft Publishers"
- },
- {
-  "Organization": "Prometheus Research"
- },
- {
-  "Organization": "Public Library of Science"
- },
- {
-  "Organization": "Research Data Alliance"
- },
- {
-  "Organization": "Riffyn"
- },
- {
-  "Organization": "SPARC"
- },
- {
-  "Organization": "SciCrunch"
- },
- {
-  "Organization": "Sense About Science USA"
- },
- {
-  "Organization": "Society for Personality Assessment (SPA)"
- },
- {
-  "Organization": "Society for Personality and Social Psychology"
- },
- {
-  "Organization": "Society for a Science of Clinical Psychology"
- },
- {
-  "Organization": "Society of Australasian Social Psychologists"
- },
- {
-  "Organization": "Software Sustainability Institute"
- },
- {
-  "Organization": "Sunlight Foundation"
- },
- {
-  "Organization": "The Odum Institute for Research in the Social Sciences"
- },
- {
-  "Organization": "Time-series Experiments in Social Sciences"
- },
- {
-  "Organization": "Ubiquity Press"
- },
- {
-  "Organization": "Wiley"
- },
- {
-  "Organization": "Zenodo"
- },
- {
-  "Organization": "rOpenSci"
- }
-]
\ No newline at end of file
+[

```

Comment:
JSON files were provided this way by Community, so this seems to be the easiest way for them to format the information. Could definitely reformat after the fact, but that does seem maybe overkill.